aws/resource_aws_redshift_cluster_test.go

```diff
@@ -342,6 +342,47 @@ func TestAccAWSRedshiftCluster_updateNodeCount(t *testing.T) {
 					testAccCheckAWSRedshiftClusterExists("aws_redshift_cluster.default", &v),
 					resource.TestCheckResourceAttr(
 						"aws_redshift_cluster.default", "number_of_nodes", "2"),
+					resource.TestCheckResourceAttr(
+						"aws_redshift_cluster.default", "cluster-type", "multi-node"),
```

Comment:
For all uses in the testing here, the `cluster-type` attribute should be `cluster_type` and `node-type` should be `node_type` to match the resource schema, otherwise testing fails. e.g.:
```
=== RUN   TestAccAWSRedshiftCluster_updateNodeCount
--- FAIL: TestAccAWSRedshiftCluster_updateNodeCount (2421.66s)
    testing.go:513: Step 1 error: Check failed: Check 3/4 error: aws_redshift_cluster.default: Attribute 'cluster-type' not found
=== RUN   TestAccAWSRedshiftCluster_updateNodeType
--- FAIL: TestAccAWSRedshiftCluster_updateNodeType (923.17s)
    testing.go:513: Step 0 error: Check failed: Check 2/2 error: aws_redshift_cluster.default: Attribute 'node-type' not found
```

Reply:
Got it, I was having issues getting the test to run locally, so I missed that, thanks!